validator/client/validator.go

```diff
@@ -937,6 +937,11 @@ func (v *validator) logDuties(slot types.Slot, duties []*ethpb.DutiesResponse_Du
 
 // UpdateFeeRecipient calls the prepareBeaconProposer RPC to set the fee recipient.
 func (v *validator) UpdateFeeRecipient(ctx context.Context, km keymanager.IKeymanager) error {
+	// only used after Bellatrix
+	if slots.ToEpoch(slots.CurrentSlot(v.genesisTime)) < params.BeaconConfig().BellatrixForkEpoch {
+		log.Info("Please plan for using the fee recipient flags post Bellatrix Hard Fork to receive transaction fee rewards on validator work.")
```

Comment:
I think we could make a case for Warnln too

Reply:
was thinking about this condition, I might need to trigger this on bellatrix hard fork again maybe in the runner.go otherwise the fee recipient might not be set when the fork happens without a restart which wouldn't be great.🤔

Reply:
I edited the warning and condition to live inside of the nil check. if the user did use the flag prior to bellatrix we should set it anyways and it will just be used after it forks. just made the error message match better.